### DISCUSSION TITLE
Questions about the checkout system and the Asset Lookup

### DISCUSSION THREAD

**Discussion by nicokempe** [snipe/snipe-it#11621] [Q&A]
I am currently using Snipe-IT on v4.9.2. Two things are currently disturbing me: 
1. The asset lookup is pretty bad on the top search bar while the bottom one (screenshot) is optimal. What is the top search bar for if not for the exact same reason on this page? Is it because the search bar is for the entire system and the automated sort in the table would be user-unfriendly or something?
2. When I checkout or checkin items, the status is not changed automatically. Sometimes you don't think of the wanted status change. Is there a configuration for this problem?

If the current stable version already has this features or if these things are wanted for, I would be thankful if someone could explain it to me.

![screenshot](https://user-images.githubusercontent.com/50241630/183018003-26e33ac8-d2d5-49f0-8f24-fbe2112a50b4.png)

**Comment by Delirium-Sama** [comment]
In my sense, the search bar you can check for any property to bring every asset with this property (exemple display port 2) if you search with the asset tag, you'll get only the assets with this tag. But yeah after that, you can search on the search bar with the asset tag haha
